Modules/xASL_module_Population.m

```diff
@@ -102,11 +100,7 @@
     SusceptPath = fullfile(x.D.TemplatesStudyDir,['MaskSusceptibility' x.S.TemplateNumberName '_bs-mean.nii']);
 
     if ~xASL_exist(SusceptPath, 'file')
-        if x.dataset.nSubjects<6 % in this case, a group template doesn't make sense and no warning is required
-            fprintf('Susceptibility mask template was missing...\n');
-        else
-            warning('Susceptibility mask template was missing...');
-        end
+        warning('Susceptibility mask template was missing...');

```

Comment:
It does not make sense to have this check here. The susceptibility mask is created only in function on line 125. 
So I would delete the entire code here from line 102 to 115. And put all reporting to the function for creating the analysis mask.

Reply:
True. I did not change this (potentially pre-existing mistake). I only changed the behavior for having too few subjects.

Reply:
Agree. Renaming helps and I did it.